Fix the Microsoft Graph Security documentation (#38898)
demisto/content: Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/README.md

For more details about the authentication used in this integration, see [Microsoft Integrations - Authentication](https://xsoar.pan.dev/docs/reference/articles/microsoft-integrations---authentication)  
*Note*: [The eDiscovery](#ediscovery-commands) and [Threat Assessment](#threat-assessment-commands) commands are only supported when using the `Authorization Code flow` with `Delegated (work or school account)` permission type.

> When using the `Authorization Code flow` for this integration, the user needs to log in as an administrator or a user with administrative privileges (`Security Reader` or `Security Administrator`) after running the **msg-generate-login-url** command and the login window appears. For more information, see [here](https://learn.microsoft.com/en-us/graph/security-authorization).

Reviewer: ```suggestion
When using the `Authorization Code flow` for this integration, you should log in as an administrator or a user with administrative privileges (`Security Reader` or `Security Administrator`) after running the ***msg-generate-login-url*** command and the login window appears. For more information, see [here](https://learn.microsoft.com/en-us/graph/security-authorization).
```
